Admin/AdminInterface.php

```diff
@@ -937,6 +937,7 @@ public function getLabel();
     public function getPersistentParameters();
 
     /**
+     * NEXT MAJOR: remove this signature
```

Comment:
I was thinking of the user as a caller, not as an implementor.

And by the way, I don't think anyone implements this interface, it's way too big.

But in any case, I don't think people are actually relying on this method, so it might be ok to remove it now. I'll do it.